Sources/Tests/FeatureTests/pfPasswordStoreTest/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+include_directories(${GTEST_INCLUDE_DIR})
+include_directories(${gtest_SOURCE_DIR}/include ${gtest_SOURCE_DIR})
+include_directories(${CMAKE_SOURCE_DIR}/Sources/Plasma/CoreLib)
+include_directories(${CMAKE_SOURCE_DIR}/Sources/Plasma/NucleusLib)
+include_directories(${CMAKE_SOURCE_DIR}/Sources/Plasma/FeatureLib)
+
+if(HAVE_LIBSECRET)
+    add_definitions(-DHAVE_LIBSECRET)
+endif(HAVE_LIBSECRET)
```

Comment:
With our, admittedly unstated, goal of modernizing our CMake, we should probably be using [target_compile_definitions](https://cmake.org/cmake/help/latest/command/target_compile_definitions.html) for this.